datarepo-clienttests/src/main/java/runner/config/SpecificationInterface.java

```diff
@@ -1,7 +1,19 @@
 package runner.config;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 public interface SpecificationInterface {
   void validate();
 
-  void display();
+  default String display() {
+    try {
+      // use Jackson to map the specification object to a JSON-formatted text block
+      ObjectMapper objectMapper = new ObjectMapper();
+      return objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(this);
+    } catch (JsonProcessingException jpEx) {
+      throw new RuntimeException(
+          "Error converting SpecificationInterface object to a JSON-formatted string");
```

Comment:
I suggest printing out the value as a string for debugging purposes.

Reply:
Done.